extensions/wikia/AdEngine/js/AdTracker.js

```diff
@@ -22,6 +22,10 @@ define('ext.wikia.adEngine.adTracker', [
 			extraParams.pf = win.ads.runtime.pf.blocking ? 'yes' : 'no';
 		}
 
+		if (win.ads && win.ads.runtime.bab && typeof win.ads.runtime.bab.blocking === 'boolean') {
```

Comment:
Could you please remove logic related to SourcePoint from this file (lines 17-19)? We've removed SP integration and most likely we forgot about this file

Reply:
I disabled init function, rest of cleaning in separate task ADEN-6587.